lib/averages.js

```diff
@@ -0,0 +1,24 @@
+function total(array) {
+  return (array.reduce(sum, 0));
```

Comment:
This isn't running for me - I see what you're trying to do, think the syntax is just off a bit for the JS reduce.